src/client/javascripts/site-details-map/site-data-loader.test.js

```diff
@@ -45,6 +45,68 @@ describe('SiteDataLoader', () => {
 
       expect(() => siteDataLoader.loadSiteDetails()).toThrow()
     })
+
+    test('should load data from map element data attribute when provided', () => {
+      const siteDetails = { coordinatesType: 'coordinates', coordinates: {} }
+      const mockMapElement = {
```

Comment:
minor suggestion but simpler / better coverage to create an element and pass it in?